.test-infra/kubernetes/kafka-cluster/04-outside-services/outside-1.yml

```diff
@@ -24,6 +24,5 @@ spec:
   ports:
   - protocol: TCP
     targetPort: 9094
-    port: 32401
-    nodePort: 32401
+    port: 32400
```

Comment:
Oh wait, actually why do we need these? Is there potential to break the existing Jenkins jobs? If so, we should disable them as part of this change

Reply:
Yes, i forgot to add that,  it will break jenkins job and we should disable it with this PR. 

@magicgoody please remove the .groovy file

Reply:
.test-infra/jenkins/job_PerformanceTests_xlang_KafkaIO_Python.groovy was removed
related Jenkins job was disabled